src/controllers/users/index.js

```diff
@@ -1,14 +1,23 @@
 'use strict';
 
 const express = require('express');
+const AuthMiddleware = require('../../middleware/AuthRequiredMiddleware');
+const {userNameMatch} = require('../../middleware/UsernameMatchRequired');
 const userHandler = new (require('../../handlers/users/UserHandler'))();
 
 let router = express.Router();
 
-router.get('/', userHandler.getUsers);
-router.put('/find', userHandler.getUserByUsername);
-router.post('/', userHandler.createUser);
-router.put('/', userHandler.updateUser);
-router.put('/delete', userHandler.deleteUserByUsername);
+router
+  .get('/', userHandler.getUsers);
+router
+  .put('/find', userHandler.getUserByUsername);
+router
+  .post('/', userHandler.createUser);
```

Comment:
What kind of authorization should be checked for on this action? 
Internal client access token or something like that maybe? I am really not sure...